Jenkinsfile

```diff
@@ -46,19 +46,34 @@ pipeline {
 		ANDROID_NDK = ""
 
 		PYTHONUNBUFFERED = "true"
+
 		JACOCO_XML = "Paintroid/build/reports/coverage/debug/report.xml"
+		JACOCO_UNIT_XML = "Paintroid/build/reports/jacoco/jacocoTestDebugUnitTestReport/jacocoTestDebugUnitTestReport.xml"
```

Comment:
I had the intention to replace all 'module'-occurrences with the variable GRADLE_PROJECT_MODULE_NAME so the Jenkinsfiles are more easily comparable between Paintroid and Catroid.
For consistency's sake we should either replace Paintroid with ${env.GRADLE_PROJECT_MODULE_NAME} or vice-versa and ditch the variable entirely. If we go for the latter option, I would suggest this could be another commit/PR which takes the restructured environment section from Catroid as template, because it includes a slightly improved documentation for the developer about the environment settings defined here.